tests/flow/test_all_shortest_paths.py

```diff
@@ -0,0 +1,86 @@
+import redis
```

Comment:
`GRAPH.QUERY g "MATCH (a), (b) RETURN allShortestPaths((a)-[:R*]->(b))"`
seems to be crashing the server

Reply:
:+1: Good catch, thanks! Fixed now.